tests/idris2/pkg003/run

```diff
@@ -0,0 +1,9 @@
+$1 --build testpkg.ipkg
+rm -f  build/
+$1 --build testpkg.ipkg --quiet
+$1 --build testpkg.ipkg --verbose
+$1 --build testpkg.ipkg --timing
+$1 --build testpkg.ipkg --codegen gambit
+$1 --build testpkg.ipkg --ide-mode
+$1 --build malformed-package-name
+$1 --build non-existent-package.ipkg
```

Comment:
Shouldn't you `rm -rf build` at the end too to cleanup after yourself?

Reply:
that would be good.